assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -76,10 +104,11 @@ class Draggable extends Component {
 	 * @param  {Object} event The non-custom DragEvent.
 	 */
 	onDragOver = ( event ) => {
+		const { onNeighborHover, blockName } = this.props;
 		const top = parseInt( this.cloneWrapper.style.top ) + event.clientY - this.cursorTop;
 
 		// Don't allow the CTA button to go over its top limit.
-		if ( 'amp/amp-story-cta' === this.props.blockName ) {
+		if ( isCTABlock( blockName ) ) {
```

Comment:
It seems a bit confusing that this limitation is also in place when dragging over another page.

Could this be changed so that the limitation is lifted when dragging over a neighbor? Although this would mean there could be a jump when going back to the current page... So not sure if that would make it better.

Reply:
Is this limitation because a if you might drag a CTA onto another page that already has one?

Reply:
The height limitation is because CTA blocks are only allowed to be placed in the bottom part of a page.

Reply:
Here's a demo video explaining the dragging situation with the CTA block: https://cloudup.com/cUthP9YKEst

Reply:
Here is a screencast of me dragging between pages. 
https://www.youtube.com/watch?v=Jocbj5tNa8I

This behaviours as I would expect. One thing I noticed that when you hover over the first page, it doesn't show any sign the CTA to isn't going to dragged. Maybe make the border of that page red. Just an idea..